moshi/src/main/java/com/squareup/moshi/Moshi.java

```diff
@@ -171,6 +172,12 @@ public Builder add(Object adapter) {
       return add(AdapterMethodsFactory.get(adapter));
     }
 
+    /** Add all factories enclosed in the provided moshi instance. */
+    public Builder add(Moshi moshi) {
```

Comment:
... the drawback of `newBuilder()` is it limits where you can add new factories. Order matters here.